client-v2/src/constants/fronts.js

```diff
@@ -6,3 +6,5 @@ export const frontStages = {
   draft: 'draft',
   live: 'live'
 };
+
+export const detectPressFailureMs = 10000;
```

Comment:
Is this what we were using in v1?

Reply:
Yes, just copied over most of the logic from v1 as users seem happy with it.